### PR TITLE
Add Oracle.ManagedDataAccess.Core package to external dependencies 

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -178,7 +178,7 @@
     <NETStandardLibrary20PackageVersion>2.0.1</NETStandardLibrary20PackageVersion>
     <NewtonsoftJsonBsonPackageVersion>1.0.1</NewtonsoftJsonBsonPackageVersion>
     <NewtonsoftJsonPackageVersion>10.0.1</NewtonsoftJsonPackageVersion>
-    <OracleManagedDataAccessPackageVersion>12.2.1100</OracleManagedDataAccessPackageVersion>
+    <OracleManagedDataAccessCorePackageVersion>2.12.0-beta2</OracleManagedDataAccessCorePackageVersion>
```
Comment (smitpatel) [inline build/dependencies.props]: @Eilon - 3rd party dependency update. Used in sample provider only. Non-shipping package.

**Comment by smitpatel** [comment]
Do we need OSS approval? It is not used in product code.
cc: @ajcvickers

**Comment by ajcvickers** [comment]
@smitpatel Yep, we do.

File: build/dependencies.props (hunk 1 above)
Comment (Eilon) [reply build/dependencies.props]: Working on it

Comment (Eilon) [reply build/dependencies.props]: Chatted with @smitpatel and @pakrym . This is a proprietary license so I'll have to review more closely when I'm back at the office on Monday.

**Comment by Eilon** [comment]
Chatted with @ajcvickers and @smitpatel and we won't add the Oracle dependency to the build, but we'll try it as a one-off to see that everything works.

**Pull request closed without merging**